ui/src/markdown.tsx

```diff
@@ -153,7 +118,8 @@ export const Markdown = ({ source }: { source: S }) => {
         return false
       }
     }
-  return <div onClick={onClick} className={clas(css.markdown, 'wave-markdown')} dangerouslySetInnerHTML={{ __html: html }} />
+  React.useEffect(() => { import('./markdown.css') }, [])
```

Comment:
Did you check if this is not downloaded twice when 2 markdown cards are present?

Reply:
It's downloaded only once.

![image](https://github.com/h2oai/wave/assets/23740173/58a1da18-5448-461d-a50c-5c085200dafd)